tests/chainer_tests/functions_tests/math_tests/test_trigonometric.py

```diff
@@ -50,16 +50,7 @@ def test_sin_forward_gpu(self):
         self.check_forward_gpu(F.sin, numpy.sin)
 
     def check_backward(self, op, x_data, y_grad):
-        x = chainer.Variable(x_data)
-        y = op(x)
-        y.grad = y_grad
-        y.backward()
-
-        func = y.creator
-        f = lambda: func.forward((x.data,))
-        gx, = gradient_check.numerical_grad(f, (x.data,), (y.grad,))
-
-        gradient_check.assert_allclose(gx, x.grad)
+        gradient_check.check_backward(op, (x_data,), y_grad)
```

Comment:
Use `x_data` instead of `(x_data, )` (second argument need not to be a tuple when it consists of one element.